backend/src/paymaster/index.ts

```diff
@@ -121,6 +121,29 @@ export class Paymaster {
     }
   }
 
+  async removeWhitelistAddress(address: string[], paymasterAddress: string, bundlerRpc: string, relayerKey: string) {
+    try {
+      const provider = new providers.JsonRpcProvider(bundlerRpc);
+      const paymasterContract = new ethers.Contract(paymasterAddress, abi, provider);
+      const signer = new Wallet(relayerKey, provider)
+      for (let i = 0; i < address.length; i++) {
+        const isAdded = await paymasterContract.check(signer.address, address[i]);
+        if (!isAdded) {
+          throw new Error(`${address[i]} is not whitelisted`)
+        }
+      }
+      const encodedData = paymasterContract.interface.encodeFunctionData('removeBatchToWhitelist', [address]);
+      const tx = await signer.sendTransaction({ to: paymasterAddress, data: encodedData });
+      await tx.wait();
+      return {
+        message: `Successfully removed whitelisted addresses with transaction Hash ${tx.hash}`
+      };
+    } catch (err: any) {
+      if (err.message.includes('is not whitelisted')) throw new Error(err);
+      throw new Error('Error while submitting transaction');
```

Comment:
Can you make this error more descriptive please? We need to get out of this habit of telling the user / developer nothing that would help them

Reply:
yes so the only failure it can get is that when the wallet does not have enough funds for the transfer but it could also be because of gas price on that chain or on the bundler rpc url so apart from those it wont be any other failure thats why wrote like that

Reply:
If that's the case them please change the error message to state that the user might not have enough funds or the gas price is currently too high - and that they should contact support if they need further assistance